packages/api/src/server.ts

```diff
@@ -2,14 +2,25 @@ import { ApolloServer } from 'apollo-server'
 import typeDefs from './typeDefs'
 import { DIRECTIVES } from '@graphql-codegen/typescript-mongodb'
 import resolvers from './resolvers'
-import { Repositories } from './types'
+import { ConfigByAddress, FeedInfo, Repositories } from './types'
 
 export async function createServer (
-  repositories: Repositories
+  repositories: Repositories,
+  config: Array<FeedInfo>
 ): Promise<ApolloServer> {
   return new ApolloServer({
     typeDefs: [DIRECTIVES, typeDefs],
     resolvers,
-    context: () => repositories
+    context: () => {
+      const configByAddress: ConfigByAddress = config.reduce(
+        (acc, feedInfo) => ({
+          ...acc,
+          [feedInfo.address]: feedInfo
+        }),
+        {}
+      )
```

Comment:
This could be problematic if someday we have singleton contracts (e.g. feeds that have the same address on multiple networks). What do you think? Is it worth it adding the network as a prefix in the keys of this object? And if so, do you want to do it now or in a separate issue?

Reply:
It's an easy improvement, I'm going to add it to this pr